src/ui/public/doc_table/doc_table.html

```diff
@@ -20,6 +20,12 @@
           class="discover-table-row"></tr>
       </tbody>
     </table>
+    <div ng-if="hits.length === sampleSize && page.last" class="discover-table-footer">
+      <center>
```

Comment:
Ah, yes definitely. Centering should be done in CSS, preferably with flexbox, but using `text-align: center` is also an option. In this case, I'd check the `.discover-table-footer` class to see if it makes sense to add these styles to that class. If there are situations where we don't want to center it's content, then create a new class, e.g.`.discover-table-footer-prompt`, and apply the style there. The updated markup would look like this:

```html
<div ng-if="hits.length === sampleSize && page.last" class="discover-table-footer">
  <div class="discover-table-footer-prompt">
    These are the first {{sampleSize}} documents matching
    your search, refine your search to see others.
  </div>
</div>
```